.circleci/config.yml

```diff
@@ -34,8 +34,7 @@ jobs:
 
   build_doc:
     docker:
-      # Pin to 3.8.16 to avoid https://github.com/mlflow/mlflow/issues/8635
-      - image: cimg/python:3.8.16
```

Comment:
https://hub.docker.com/layers/cimg/python/3.8.16/images/sha256-9ab6ff6b482aef7c157bf1540ff661cbed0586f3b11f6cce20a3e751469367f0?context=explore

hasn't been updated for two years. not sure what has changed.